docs/explore.md

```diff
@@ -11,7 +11,7 @@ There are two distinct aspect to system programming:
   - Can treat memory directly as bits and bytes
   - Little or no abstraction on the hardware
 
-To implement many low-level systems, for instance a memory manager, you need raw access.
+To implement many low-level systems (for instance, a memory manager), you need raw access.
```

Comment:
If you're going to correct the punctuation here, it's probably also a good idea to correct the vocabulary.  'For instance' is informal and should be 'for example' in formal writing.

Reply:
OK, I'll change 'for instance' to 'for example'.
Rather, I found 'many low-level systems' vague here; 'a low-level system of various kinds' may be clearer?